nextflow_schema.json

```diff
@@ -73,7 +88,8 @@
                 },
                 "single_end": {
                     "type": "boolean",
-                    "description": "If data is single-ended Illumina reads instead of paired-end"
+                    "description": "If data is single-ended Illumina reads instead of paired-end",
+                    "help_text": "When using a sample sheet with `--input` containing forward an reverse reads, specifying `--single_end` will only extract forward reads and treat the data as single ended instead of extracting forward and reverse reads."
```

Comment:
```suggestion
                    "help_text": "When using a sample sheet with `--input` containing forward and reverse reads, specifying `--single_end` will only extract forward reads and treat the data as single ended instead of extracting forward and reverse reads."
```